src/gsy_e/gsy_e_core/sim_results/endpoint_buffer.py

```diff
@@ -203,10 +207,11 @@ def _read_market_stats_to_dict(market: "MarketBase") -> Dict:
         stats_dict["market_fee"] = market.market_fee
         stats_dict["const_fee_rate"] = (market.const_fee_rate
                                         if market.const_fee_rate is not None else 0.)
-        stats_dict["feed_in_tariff"] = GlobalConfig.FEED_IN_TARIFF
+        stats_dict["feed_in_tariff"] = get_feed_in_tariff_rate_from_config(market)
         stats_dict["market_maker_rate"] = get_market_maker_rate_from_config(market)
         return stats_dict
 
+    # pylint: disable=too-many-branches
```

Comment:
Or remove the class-level warning in favor of this one. 

On another note, from what I see here Pylint has a point, this should in reality be refactored to a dispatching class / module since most of the if / elif blocks do not access state from the class and can be pure functions.

Reply:
I would prefer get it dealt with later with an small refactoring ticket as this PR targeted to solve bug also that its one of blocker to deploy

Reply:
I do not think that this needs a ticket, an afterburner PR after this one would be sufficient

Reply:
Okay will handle this and future market handling for feedin tariff as after burner PR